container/docker/handler.go

```diff
@@ -242,17 +273,22 @@ func (self *dockerContainerHandler) getFsStats(stats *info.ContainerStats) error
 	if self.ignoreMetrics.Has(container.DiskUsageMetrics) {
 		return nil
 	}
+	var device string
 	switch self.storageDriver {
+	case devicemapperStorageDriver:
```

Comment:
Apologies If I wasn't clear when we discussed in person @pmorie.

Alternative option `1` you proposed makes sense. Let's do that.